.travis.yml

```diff
@@ -10,7 +10,12 @@ cache:
 # at $HOME/miniconda/pkgs. That is a win-win because when re-creating the
 # env we will download only the new packages.
 before_cache:
-  - rm -rf $HOME/miniconda/envs/IOOS
+  - rm -rf $HOME/miniconda/envs/BiG-CZ
+  - rm -rf $HOME/miniconda/envs/PAGE
```

Comment:
We want to cache the packages not the envs, that way we save time when the packages are the same by downloading only what is new.